### DISCUSSION TITLE
FIP0036 & FIL Poll

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/FIPs#464] [Announcements]
[FIP0036 - Introducing a Sector Duration Multiple for Longer Term Storage Commitment](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0036.md) was first introduced to the Filecoin community in June 2022. 

The FIP immediately sparked debate and discussion across the Filecoin ecosystem, leading to more engagement in community governance processes than ever before! 

Since the FIP draft was first published, the community generated: 
*  More than 500 comments across [three discussion threads](https://github.com/filecoin-project/FIPs/discussions?discussions_q=label%3AFIP0036); 
*  [5 different FIP drafts](https://github.com/filecoin-project/FIPs/pulls?q=is%3Apr+is%3Aclosed), each iterating on previous versions; 
*  Discussions held in community meetings, including [Core Devs](https://github.com/filecoin-project/tpm#core-devs-meetings) and Storage Provider Working Groups, and; 
*  Public learning sessions, including a variety of author panels and [community AMAs](https://twitter.com/FilecoinSP/status/1567974802266390529?s=20&t=VQRLJfm-xmzjs4wITxgzHA). 

Increased community engagement with governance proposals is a critical indicator of ecosystem health. However, this growth also poses new challenges.  As more voices and perspectives are considered, it becomes more challenging to legitimately gauge consensus, particularly across different stakeholder groups. 

**As a result, stewards of Filecoin governance stewards have decided to open up [FIL Poll](https://filpoll.io/about) for community voting on FIP0036.** 

Please continue reading for detailed information about the FIL Poll process, as well as instructions on how to cast your vote. 

---
## Summary of FIP0036
The current draft of [FIP0036](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0036.md) is considered final; as we prepare for a community vote, no further protocol changes to the draft will be accepted. 

The current version of FIP0036, which community members will be asked to 'approve' or 'reject', proposes the following: 
* Increase minimum sector duration time from six months to 1 year 
* Increase the maximum sector duration time from 1 years to 3.5 years 
* Introduce a Sector Duration Multiplier for all sectors
* Increase Sector Initial Consensus Pledge multiplier from 30% to 50% 

FIP0036 further specifies that these changes, once implemented, will apply to new sectors as well as during sector upgrade and extension. 

In addition, the increases in sector minimum and maximum durations will increase gradually.  Beginning at implementation, duration maximum and minimum periods will increase by six months every week for four weeks until the goal duration is reached. 

Core Devs and the Filecoin Plus (Fil+) governance community have each confirmed that existing sectors with FiL+ deals will be eligible for sector extension and the commensurate multiplier adjustment. Notably, Fil+ deals themselves cannot be extended. 

Lastly, Core Devs have confirmed that, if approved, FIP0036 will be scheduled for inclusion in the network v17 upgrade.  The upgrade is currently scheduled for early November, and will also include FIPs [0029](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md), [0034](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0034.md), [0041](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0041.md), [0044](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0044.md) (pending approval), and [0045](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0045.md) (pending approval).

Core Devs have also indicated that [FIP0047](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md) may be a security prerequisite for implementing FIP0036.  If FIP0036 were to be approved, FIP0047 may also be reconsidered for inclusion (pending approval) in nv17.  

For more information on the nv17 upgrade, see [HERE](https://github.com/filecoin-project/tpm/discussions/91). 

## The FIL Poll Governance Process 
Like other open-source protocols, most proposals in the Filecoin ecosystem do not receive a community vote. 

Instead, proposals are judged according to 'soft consensus', where open design, public documentation, and various rounds of iteration and technical auditing allows community members to participate deeply on topics of interest.

However, FIP0036 proposes changes to Filecoin cryptoeconomics, making it a complex and high-impact proposal that does not benefit from the same mechanism of consensus. 

As such, the FIP0036 ['Last Call' period](https://github.com/filecoin-project/FIPs/discussions/314) will begin on Wednesday, September 14 at 3:00pm PST. **During 'Last Call', community members are welcome to vote to 'approve' or 'reject' FIP0036 in entirety.**

### When is voting?

Voting begins at 3:00pm PST on Wednesday, September 14. 

FIL Poll will remain open until 3:00pm PST on Wednesday, September 28. 

Voting will be open and ongoing during the entire two week 'Last Call' period.  Any person can cast their vote at any time within that window. 
### Who gets to vote? 
The Filecoin ecosystem does not 'gate' who is and is not a member of the community.  It is an open and decentralized ecosystem, and we likewise welcome anyone who wishes to vote. 

To ensure that the voting process remains secure and robust, votes will be organized and weighted within the following categories: 

**1. Storage Providers who make storage deals** (vote weighted by deal bytes, you may sign by either worker address or owner address)
**2. Storage Providers who provide storage capacity** (weighted by raw bytes,  you may sign by either worker address or owner address)
**3. Storage Clients** (weighted by deal bytes)
**4. Token Holders** (weighted by FIL balance for both regular and multisig wallets, and miner actor's locked reward)
**5. Core Devs** (weighted by headcount) 

The above categories represent the broadest swath of community members, and provide a weighted voting constituency for anyone who would like to participate. 

Storage provider votes will be casted to two separate groups, indicating the two separate functions that they predominantly serve in the network.  The decision to segment storage provider votes by raw bytes and deal bytes further reflects the impact that this FIP will specifically have within the storage provider community.  

It is the intent of Filecoin governance stewards that this weighting schema- particularly for storage providers- allows for the fairest possible voting patterns to emerge.  

### How will consensus be reached? 

**Voting on FIP0036 asks community members to either Approve or Reject the FIP.**

In order to be approved, 3/5 voting groups must 'Approve' the FIP with a simple majority (>50%). 

If 75% or more of one or multiple groups votes to reject the FIP, it will automatically be vetoed (i.e., rejected). 

There is no minimum threshold or quorum for voting. All constituencies are valid so long as they achieve a single vote. 

### What happens after the vote?
During the vote process, real-time results will be publicly available. **However, consensus will not be determined until polling data is processed.**  This will happen immediately after the vote closes. 

Data processing is necessary to ensure no voting anomalies are present.  It is also necessary in order to separate deal-making and capacity -providing storage provider constituency groups.  During live polling, all storage provider votes will be tallied in a single "storage provider" category.  

**Data processing will be fully transparent and will confirm the results of the poll.**  The model used for post processing data will be made publicly available. All raw polling data will likewise be made available. 

Once polling is complete, results will be considered against the abovementioned consensus criteria and an outcome to approve or reject the FIP will be made. 

If the FIP is approved, it will automatically be scheduled for inclusion in nv17. 

If the FIP is rejected, it will no longer be considered eligible (as currently specified) for any future upgrades. 

The final decision will be reported on the #fil-fips channel on Slack, and documented more thoroughly on the [Governance Forum](https://github.com/filecoin-project/FIPs/discussions) on Github. 

### How do I vote?

Community members can utilize Ledger (via Glif) or their Lotus client for voting.  For detailed instructions, see [HERE](https://pl-strflt.notion.site/How-to-sign-a-vote-on-FilPoll-using-Lotus-Glif-95d9b0a32f9c48858574f9cb072c054b).

For help, you are welcome to ask questions at any time in the #fil-fips channel in Slack.

**Comment by jennijuju** [comment]
## FAQ

### If I am an SP, which account should I sign the vote with?

You can use either worker address or the owner address. The system will then find all the miner actors that using that account as owner/worker, and count in the power bytes accordingly (See `Who gets to vote?` above).

Note: The balance of the worker/owner address you used to sign, and the **balance of the miner actor's available balance** will also be added to the `Token Holders` count.

> DEPRECATED ON SEP 20,2022 Note: The balance of the worker/owner address you used to sign, and the **balance of the miner actor's locked reward** will also be added to the `Token Holders` count. DEPRECATED

### I am an SP and my owner account is a multisig, how should I sign my vote?**
   
You can sign the vote with the multisig-owner address of a miner actor, but the vote is only valid when the number of signatures received matches the msig's threshold. We recommend storage providers with multisignature owners to use their worker-address instead, for ease of voting.

### I am a client, and I'd like to vote, however FilPoll suggests the group is not supported. What should I do?

As mentioned in `Who gets to vote?`,  storage clients' votes will be collected (weighted by deal bytes) during the post-polling data process.  Simply sign the vote using the account you made deals with, and the deal bytes sum (valid deals @ epoch 2162760) will be added to the `Storage Client` group during the post-polling data process. for vote counting.

Note: The balance of the address you used to sign, will also be added to the `Token Holders` counting.

### Why the vote result is not shown on the FilPoll live?

You can see the accounts that have voted live on FilPoll. However, the current FilPoll vote counting mechanism has some restrictions. That is why the post-polling data process will be performed to capture the broadest swath of community members and provide a weighted voting constituency for anyone who would like to participate.
**All data processing will be immediate and fully transparent. The model used for post processing will be made available before the vote closes. All raw polling data will likewise be made available.** We welcome community members to process and/or validate the poll result as well! We will also be improving FilPoll gradually to provide a better polling experience for the Filecoin ecosystem

**Comment by whyrusleeping** [comment]
Heads up that ledgers don't work with filpoll, is there another way we can vote with tokens in a hardware wallet?

**Reply by jennijuju** [reply]
please follow the instructions [here](https://pl-strflt.notion.site/How-to-sign-a-vote-on-FilPoll-using-Lotus-Glif-95d9b0a32f9c48858574f9cb072c054b)  - the `Sign vote with a Ledger wallet using glif` session. If you have any more qs - please ask in #fil-fips 💙

**Reply by whyrusleeping** [reply]
Ah, seems like it just requires the latest version of the filecoin app on ledger.

**Comment by jennijuju** [comment]
## Group 5 - Core Dev

op by @kaitlin-beegle  - [source](https://github.com/filecoin-project/tpm/discussions/113#discussion-4408546)

After discussion, it was decided that anyone included in the #fil-core-devs channel on Filecoin project Slack as of mid-August (approximately) would be eligible to vote as a Core Dev during the FIP0036 FIL Poll.

Currently, those people include:

Hubert Bugaj (Forest, Chainsafe)
Aayush Rajasekaran (Lotus, Protocol Labs)
Afri Schoedon (Chainsafe)
Amer Ameen (Chainsafe)
Alex North (CryptoNetLab, Protocol Labs)
David Ansermino (Chainsafe)
David Himmelstrup (Forest, Chainsafe)
Dragan Zurzin (Protocol Labs)
Guillaume Potier (Forest, Chainsafe)
Brooklyn Zelenka (Fission)
Geoff Stuart (Lotus, Protocol Labs)
Honghao Qiu (Protocol Labs)
Juan Benet (Protocol Labs)
Jennifer Wang (Lotus, Protocol Labs)
Josh Jones (Forest, Chainsafe)
Joss Hua (Venus, IPFSForce)
Jakub Sztandera (Protocol Labs)
Kaitlin Beegle (Filecoin Foundation
Lee Raj (Forest, Chainsafe)
Lukasz Magiera (Protocol Labs)
Molly Mackinlay (Protocol Labs)
@nemo (Protocol Labs)
@nickle (Protocol Labs)
Raul Kripalani (Protocol Labs)
Sonia John (Filecoin Foundation)
@Stebalien (Protocol Labs)
Steven Li (Venus, IPFSForce)
Vik Kalghatgi (CryptoEconLab, Protocol Labs)
Dimitris Vyzovitis (Protocol Labs)
@why (Protocol Labs)
@ZenGround0 (Protocol Labs)
ZX Zhang (CryptoEconLab, Protocol Labs)
